test/connectors/CodeLinkConnectorISpec.scala

```diff
@@ -19,7 +19,7 @@ package connectors
 import config.AppConfig
 import org.mockito.Mockito.{reset, when}
 
-class CodeLinkConnectorSpec extends ConnectorSpec {
+class CodeLinkConnectorISpec extends ConnectorISpec {
```

Comment:
This is not really an integration test (no use of wiremock) so I'd leave out the "ISpec" postfix.

Reply:
I'll change then the extended trait.